jaws-journey-deploy/orb.yml

```diff
@@ -229,6 +238,15 @@ commands:
           command: |
             include ./scripts/install_adoptopenjdk.sh
 
+  tf-install-adoptopenjdk:
```

Comment:
```suggestion
  install-adoptopenjdk:
```

Reply:
Doesn't work - the install-adoptopenjdk uses sudo - think the dockers are too different.